shpy

```diff
@@ -192,8 +203,22 @@ _shpySpySetErrorOutput() {
 
 _shpySpyPrintOutput() {
     # shellcheck disable=SC2039
-    local output
-    eval output="\$_shpy_${name}_output"
+    local call_index count output output_fixture
+
+    stub_dir=$(realpath "$_shpy_spies_dir")
+    outputs_dir=${stub_dir}/outputs/${name}
+
+    count=$(getSpyCallCount "$name")
+    call_index=$((count - 1))
```

Comment:
If `output_num` in `createSpy` started at 1, this math shouldn't be necessary here

Reply:
True, but I did it this way to be consistent with `_shpySpySaveArgs`, which saves arguments in separate files starting at 0.

Let me know if you want me to change the outputs to start at 1

Reply:
In that case, I think it makes sense to leave it at 0